src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -210,6 +217,21 @@ def create_container(cmd,
     return sdk_no_wait(no_wait, container_group_client.create_or_update, resource_group_name, name, cgroup)
 
 
+def _build_identities_info(identities):
+    identities = identities or []
```

Comment:
Also instead of building the identity_types string using str.join and hardcoded strings, you should use the [ResourceIdentityType](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-containerinstance/azure/mgmt/containerinstance/models/container_instance_management_client_enums.py) enum. You can import it from azure.mgmt.containerinstance or use cmd.get_model().

P.S. Side note but there is a model called [ContainerGroupIdentity](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-containerinstance/azure/mgmt/containerinstance/models/container_group_identity.py) that can be used to build the identity object you pass to [ContainerGroup](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-containerinstance/azure/mgmt/containerinstance/models/container_group.py). If this model makes your method more concise, consider using it.